politeiawww/cmd/pictl/proposal.go

```diff
@@ -26,6 +27,25 @@ import (
 	"github.com/decred/politeia/util"
 )
 
+const (
+	monthInSeconds      int64 = 30 * 24 * 60 * 60
+	fourMonthsInSeconds int64 = 4 * monthInSeconds
+)
+
+var (
+	// defaultStartDate is the default proposal metadata start date in
+	// Unix time. It defaults to one month from now.
+	defaultStartDate = time.Now().Unix() + monthInSeconds
```

Comment:
There is no way this is correct. This is only created once at start of day.

Reply:
After talking ti @lukebp it was explained this is always envoked and thus the time is capture every time.